### PR TITLE
Add Wano 39

### PR DESCRIPTION
**Pull request by TheIceCreamTroll**
## Describe your changes
Adds Wano 39

PR #40 *must* be merged first for the season numbers to be correct

## Checklist for submitting new .nfo files
- [x] I have removed the old .nfo files that are being replaced
- [x] I have validated that the new .nfo file works correctly in Plex
- [x] I have updated the README, fixing the list of current missing episodes
- [x] I have modified exceptions.json, removing any exceptions no longer required

image

### CODE REVIEW

File: One Pace/Season 35/One Pace - s35e39 - Fool of a Lord.nfo
Comment (SpykerNZ) [inline One Pace/Season 35/One Pace - s35e39 - Fool of a Lord.nfo:1]: This .nfo file has lower case episode/season naming: `s35e39`. would be nice to have this upper case for consistency :)

**Review by SpykerNZ — COMMENTED** [review]
minor comment around file naming. other than that looks great! Will merge this in once PR #40 is merged. Thanks!

File: One Pace/Season 35/One Pace - s35e39 - Fool of a Lord.nfo
Comment (TheIceCreamTroll) [reply One Pace/Season 35/One Pace - s35e39 - Fool of a Lord.nfo:1]: augh! could have sworn it was right when I pushed it. Nice catch!

Comment (TheIceCreamTroll) [reply One Pace/Season 35/One Pace - s35e39 - Fool of a Lord.nfo:1]: This should now be fixed

**Comment by trulow** [comment]
The season is incorrect. The file should be s34e39

**Comment by SpykerNZ** [comment]
@trulow There has been an extra episode from the One Pace team squeezed in as Season 25. All subsequent seasons have been bumped as a result to maintain parity with the One Pace episode structure.

@TheIceCreamTroll  There still seems to be some conflicts with this branch after #40 was merged in. If you could fix those up then can merge this episode in! 😄

**Comment by TheIceCreamTroll** [comment]
> @trulow There has been an extra episode from the One Pace team squeezed in as Season 25. All subsequent seasons have been bumped as a result to maintain parity with the One Pace episode structure.
> 
> @TheIceCreamTroll There still seems to be some conflicts with this branch after #40 was merged in. If you could fix those up then can merge this episode in! 😄

Rebased. Should be good to merge now!

**Comment by SpykerNZ** [comment]
sorry in the delay in getting to this. looks good, will merge. Thanks for all your help! 

Let me know if you want to be added as a contributor. Always good to have a few extra eyes on PRs!

**Comment by TheIceCreamTroll** [comment]
> sorry in the delay in getting to this. looks good, will merge. Thanks for all your help!
> 
> Let me know if you want to be added as a contributor. Always good to have a few extra eyes on PRs!

Feel free to add me as a contributer! I'll try and polish up the scripts I wrote to automate most of this process before I share them, but I've been quite busy lately, so it'll be a bit

**Comment by SpykerNZ** [comment]
@TheIceCreamTroll I have invited you to be a contributor 🎉

Sounds good regarding your scripts, no rush! :)

**Pull request merged**